README.md

```diff
@@ -0,0 +1,39 @@
+# WikidataPositionHistory
+
+Welcome to your new gem! In this directory, you'll find the files you need to be able to package up your Ruby library into a gem. Put your Ruby code in the file `lib/wikidata_position_history`. To experiment with that code, run `bin/console` for an interactive prompt.
+
+TODO: Delete this and the text above, and describe your gem
```

Comment:
Looks like you forgot to edit this file (here, and a few places below)

Reply:
I've updated that file with basic usage instructions, etc.